packages/dr.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class dr < Package
+  description 'A simple ed-like devRant client'
+  homepage 'https://github.com/Ewpratten/dr'
+  version '1.0-cros'
+  source_url 'https://codeload.github.com/Ewpratten/dr/tar.gz/1.0-chr'
```

Comment:
It would be better if you used the tagged release:
```
- source_url 'https://codeload.github.com/Ewpratten/dr/tar.gz/1.0-chr'
+ source_url 'https://github.com/Ewpratten/dr/archive/1.0-nix.tar.gz'
```